Header header
bool detect_pipe
bool detection_stable
geometry_msgs/Point32[] points
